### PR TITLE
Reader: Add 3 variations of the new Reader intro banner

### PR DESCRIPTION
**Pull request by jancavan**
This PR replaces the current Reader intro banner and adds the 3 variations which will be AB tested (in a separate PR).

**(A)**
screenshot 2017-07-11 11 21 03

**(B)**
screenshot 2017-07-11 12 00 57

**(C)**
screenshot 2017-07-11 13 09 11

### CODE REVIEW

File: client/reader/following/style.scss [hunk 1]
```diff
@@ -104,3 +103,33 @@
 		}
 	}
 }
+
+// Following intro blue
+.following__intro-blue {
```
Comment (samouri) [inline client/reader/following/style.scss]: is there old css we should be deleting at the same time

Comment (jancavan) [inline client/reader/following/style.scss]: Yes, I'll eventually be deleting the old styles.

**Comment by drw158** [comment]
I would run the svg through an svg minify tool.

**Comment by jancavan** [comment]
This is what the 3 variations currently look like:

**(A)**
screenshot 2017-07-17 10 30 09

**(B)**
screenshot 2017-07-17 10 29 18

**(C)**
screenshot 2017-07-17 10 24 22

I added back the original banner. I'll remove it once we have the AB test PR and we're ready to roll this out.

**Pull request merged**